### PR TITLE
Bugzilla 1538473 - changes to Recycling section

### PR DESCRIPTION
**Pull request by liangxia**
https://bugzilla.redhat.com/show_bug.cgi?id=1538473

This should apply to branch/enterprise-3.6 and above.

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
Work continued in https://github.com/openshift/openshift-docs/pull/7732. Closing this PR.

**Comment by tmorriso-rh** [comment]
@liangxia Please review.

**Comment by tmorriso-rh** [comment]
@liangxia Please review.

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: architecture/additional_concepts/storage.adoc [hunk 1]
```diff
@@ -168,7 +169,9 @@ spec:
     - name: nfsvol
       mountPath: /scrub
 ----
-<1> However, the particular `server` and `path` values specified in the custom recycler pod template in the `volumes` part is replaced with the particular corresponding values from the PV that is being recycled.
+<1> Namespace where the recycler pod will run. `openshift-infra` is the recommended one, as it already has `pv-recycler-controller` service account that can recycle volumes.
```
Comment (ahardin-rh) [inline architecture/additional_concepts/storage.adoc]: s/is the recommended one/is recommended

File: architecture/additional_concepts/storage.adoc [hunk 2]
```diff
@@ -168,7 +169,9 @@ spec:
...
+<2> Name of service account that is allowed to mount NFS volumes. It must exist in the namespace specified by <1>. `pv-recycler-controller` account is recommended, as it is automatically created in `openshift-infra` namespace and has all the required permissions.
```
Comment (ahardin-rh) [inline architecture/additional_concepts/storage.adoc]: I would see how the reference to <1> in the description for <2> builds. You may need to spell it out.

**Comment by ahardin-rh** [comment]
@tmorriso-rh Just minor suggestions from me! 🌸

File: architecture/additional_concepts/storage.adoc (hunk 1 above)
Comment (tmorriso-rh) [reply architecture/additional_concepts/storage.adoc]: Fixed.

File: architecture/additional_concepts/storage.adoc (hunk 2 above)
Comment (tmorriso-rh) [reply architecture/additional_concepts/storage.adoc]: Fixed.

**Comment by tmorriso-rh** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/7839.

**Pull request closed without merging**